service/provider/src/main/java/gov/va/vro/service/provider/MasPollingProcessor.java

```diff
@@ -1,42 +1,152 @@
+            .equals(MasStatus.getMasStatus(masCollectionStatus.getCollectionStatus()))) {
+          isCollectionReady = true;
+        }
+      }
+    } catch (Exception e) {
+      log.error("Error in calling collection Status API ", e);
+      throw new MasException(e.getMessage(), e);
     }
+    return isCollectionReady;
   }
 
-  private boolean checkCollectionStatus(int collectionId) {
-    return new Random().nextBoolean();
+  public AbdEvidence getCollectionAnnots(MasAutomatedClaimPayload claimPayload)
+      throws MasException {
+
+    log.info(
+        "Collection {} is ready for processing, calling collection annotation service ",
+        claimPayload.getCollectionId());
+    AbdEvidence abdEvidence = new AbdEvidence();
+    try {
+      var response =
+          masCollectionAnnotsApiService.getCollectionAnnots(claimPayload.getCollectionId());
+      for (MasCollectionAnnotation masCollectionAnnotation : response) {
+        log.info(
+            "Collection Annotation Response : Collection ID  {}",
+            masCollectionAnnotation.getCollectionsId());
+        log.info(
+            "Collection Status Response : Veteran FileId  {}  ",
+            masCollectionAnnotation.getVtrnFileId());
+
+        MasCollectionAnnotsResults masCollectionAnnotsResults = new MasCollectionAnnotsResults();
+        abdEvidence = masCollectionAnnotsResults.mapAnnotsToEvidence(masCollectionAnnotation);
+
+        log.info("AbdEvidence : Medications {}  ", abdEvidence.getMedications().size());
+        log.info("AbdEvidence : Conditions {}  ", abdEvidence.getConditions().size());
+        log.info("AbdEvidence : BP {}  ", abdEvidence.getBloodPressures().size());
+        break;
+      }
+    } catch (Exception e) {
+      log.error("Error in calling collection Status API ", e);
+      throw new MasException(e.getMessage(), e);
+    }
+    return abdEvidence;
+  }
+
+  public GeneratePdfResp generatePdf(MasAutomatedClaimPayload claimPayload, AbdEvidence abdEvidence)
+      throws MasException {
+
+    GeneratePdfPayload generatePdfPayload = new GeneratePdfPayload();
+    generatePdfPayload.setEvidence(abdEvidence);
+    generatePdfPayload.setClaimSubmissionId(claimPayload.getClaimDetail().getBenefitClaimId());
+    generatePdfPayload.setDiagnosticCode(
+        claimPayload.getClaimDetail().getConditions().getDiagnosticCode());
+    VeteranInfo veteranInfo = new VeteranInfo();
+    veteranInfo.setFirst(claimPayload.getFirstName());
+    veteranInfo.setLast(claimPayload.getLastName());
+    veteranInfo.setMiddle("");
+    veteranInfo.setBirthdate(claimPayload.getDateOfBirth());
+    generatePdfPayload.setVeteranInfo(veteranInfo);
+    log.info(
+        "Generating pdf for claim: {} and diagnostic code {}",
+        generatePdfPayload.getClaimSubmissionId(),
+        generatePdfPayload.getDiagnosticCode());
+    try {
+      log.info(generatePdfPayload.toString());
+      String response = camelEntrance.generatePdf(generatePdfPayload);
```

Comment:
I got an exception here when I tried in my local machine. But it may not be related to the work done in this PR.

2022-10-27 17:52:38.871 ERROR 1 --- [cation-exchange] g.v.v.s.provider.MasPollingProcessor     : Error in generate pdf

java.lang.Exception: RESPONSE from generatePdf returned error reason: '7401'
	at gov.va.vro.service.provider.MasPollingProcessor.generatePdf(MasPollingProcessor.java:143) ~[provider-0.1.0-SNAPSHOT.jar!/:na]
	at gov.va.vro.service.provider.MasPollingProcessor.process(MasPollingProcessor.java:45) ~[provider-0.1.0-SNAPSHOT.jar!/:na]
	at org.apache.camel.support.processor.DelegateSyncProcessor.process(DelegateSyncProcessor.java:65) ~[camel-support-3.19.0.jar!/:3.19.0]